test/unit/jasmine-boot.js

```diff
@@ -84,6 +84,7 @@ async function initializePDFJS(callback) {
       "pdfjs-test/unit/unicode_spec.js",
       "pdfjs-test/unit/util_spec.js",
       "pdfjs-test/unit/writer_spec.js",
+      "pdfjs-test/unit/xfa_parser_spec.js",
```

Comment:
Unless these unit-tests can't for some reason be run in Node.js, please don't forget to update https://github.com/mozilla/pdf.js/blob/master/test/unit/clitests.json as well :-)